.jenkinsci-new/builders/x64-linux-build-steps.groovy

```diff
@@ -132,6 +132,20 @@ def buildSteps(int parallelism, List compilerVersions, String build_type, boolea
 def successPostSteps(scmVars, boolean packagePush, String dockerTag, List environment) {
   stage('Linux success PostSteps') {
     withEnv(environment) {
+
+      // handling build time results
+      if (currentBuild.currentResult == "SUCCESS" && !params.fuzzing) {
+        sh(".jenkinsci-new/helpers/exportBuildTime.py buildTimeResult.txt")
+        zip archive: true, dir: '', glob: 'buildTimeResult.csv', zipFile: 'buildTimeMeasurement.zip'
+        archiveArtifacts artifacts: 'buildTimeMeasurement.zip'
```

Comment:
New CI can run just `clang` compiler and it will fail in `successPostSteps` because `buildTimeMeasurement.zip`  do not exist. 
so you need to check which compiler is used or just:
```suggestion
        archiveArtifacts artifacts: 'buildTimeMeasurement.zip', allowEmptyArchive: true
```